nym-vpn-lib/src/platform/android.rs

```diff
@@ -133,12 +149,9 @@ pub extern "system" fn Java_net_nymtech_vpn_NymVpnService_runVPN(_env: JNIEnv, _
     let state = RUNTIME.block_on(get_vpn_state());
     if state != ClientState::Disconnected {
         warn!("Invalid vpn state: {:?}", state);
-        return;
     }
 
-    let vpn = RUNTIME
-        .block_on(take_vpn())
-        .expect("VPN configuration was cleared before it could be used");
```

Comment:
I think the previous log was a bit better as it was more explicit in what is happening, as in this case, the VPN was initialized, but then got disconnected between the check from above and the call from here.

Reply:
Okay, we can revert.